api.js

```diff
@@ -56,7 +61,17 @@ Api.prototype._runFile = function (file) {
 		args.push('--sorted');
 	}
 
-	return fork(args)
+	var options = {
+		execArgv: []
+	};
+
+	if (this.require) {
+		this.require.forEach(function (moduleId) {
+			options.execArgv.push('--require', moduleId);
+		});
```

Comment:
I think it's time we simplify argument passing to the forked process.

@jamestalmage @vdemedes I propose we just JSON.stringify() the data we want to pass instead of using CLI args. Thoughts?